examples/with-postgres/ley.config.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config({ path: '.env.local' })
+const { parse } = require('pg-connection-string')
+
+const options = {}
+
+if (process.env.DATABASE_URL) {
+  const { host, database, user, password, ssl } = parse(
+    process.env.DATABASE_URL
+  )
+
+  process.env.PGHOST = host
```

Comment:
Did you mean to assign to `options` here?

Reply:
Yes yes, you are right. Refactoring...